src/content/docs/mobile-monitoring/new-relic-mobile-flutter/monitor-your-flutter-application.mdx

```diff
@@ -0,0 +1,431 @@
+
+        //Android Specific
+        // Optional: Enable or disable collection of event data.
+        analyticsEventEnabled: true,
+
+        // Optional:Enable or disable reporting successful HTTP requests to the MobileRequest event type.
+        networkErrorRequestEnabled: true,
+
+        // Optional:Enable or disable reporting network and HTTP request errors to the MobileRequestError event type.
+        networkRequestEnabled: true,
+
+        // Optional:Enable or disable crash reporting.
+        crashReportingEnabled: true,
+
+        // Optional:Enable or disable interaction tracing. Trace instrumentation still occurs, but no traces are harvested. This will disable default and custom interactions.
+        interactionTracingEnabled: true,
+
+        // Optional: Enable or disable capture of HTTP response bodies for HTTP error traces and MobileRequestError events.
+        httpResponseBodyCaptureEnabled: true,
+
+        // Optional: Enable or disable agent logging.
+        loggingEnabled: true,
+
+        //iOS Specific
+        // Optional: Enable or disable automatic instrumentation of WebViews
+        webViewInstrumentation: true,
+
+        //Optional: Enable or disable Print Statements as Analytics Events
+        printStatementAsEventsEnabled : true,
+
+        // Optional: Enable or disable automatic instrumentation of Http Request
+        httpInstrumentationEnabled:true
+        );
+
+    NewrelicMobile.instance.start(config, () {
+        runApp(MyApp());
+    });
+
+    class MyApp extends StatelessWidget {
+    ....
+```
+
+Make sure you pasted your application token(s) (copied from [Step 2](#app-token)) into `appToken = ""` in the code above. If you're monitoring both Android-native and iOS-native apps, you'll need to add tokens for both.
+
+## (Android-only) Step 5: Configure your Android app
+
+If you have an Android-native app, you'll need make the following changes in your Android app:
+
+1. Add the following changes to your app's `android/build.gradle` file:
+
+```dart
+buildscript {
+        ...
+        repositories {
+            ...
+            mavenCentral()
+        }
+        dependencies {
+            ...
+            classpath "com.newrelic.agent.android:agent-gradle-plugin:6.9.2"
+        }
+    }
+```
+2. Apply the `newrelic` plugin to the top of your `android/app/build.gradle` file:
+
+```dart
+apply plugin: "com.android.application"
+apply plugin: 'newrelic' // <-- add this
+```
+
+3. Add `INTERNET` and `ACCESS_NETWORK_STATE` permissions in your `AndroidManifest.xml` file:
+
+```dart
+<uses-permission android:name="android.permission.INTERNET" />
+<uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+```
+
+And you're done! You should start seeing data in New Relic within 5 minutes. 
+
+## View Flutter data in New Relic
+
+Now that you've got the Flutter agent sending data to New Relic, it's time to view, track, and set up alerts for your Flutter app data. 
+
```

Comment:
@ndesai-newrelic In our meeting you mentioned we're storing logs as custom events so we need to give instructions on how to query the logs. Can you provide a query example and I'll add a section here?